### DISCUSSION TITLE
Remove TRestxxxProcess/TRestMetadata (char *cfgFilename) constructor?

### DISCUSSION THREAD

**Discussion by jgalan** [rest-for-physics/framework#159] [General]
Inside `TRestDetectorHiitsToSignalProcess` I found the following comment.

```
// __________________________________________________________
//     TODO : Perhaps this constructor should be removed
//            since we will allway load the config from TRestRun
//            when we use AddProcess. It would be necessary only if we use the
//            process stand alone but even then we could just call LoadConfig
//            __________________________________________________________
TRestDetectorHitsToSignalProcess::TRestDetectorHitsToSignalProcess(char* cfgFileName) {
```

I am now reviewing this process and I will remove that comment. Then, if this is a real concern we could create an issue, discuss it and apply the change.

The proposal implies that if we want to use the metadata/process class stand-alone we instantiate it and call `LoadConfig`.

@rest-for-physics/core_dev

**Comment by nkx111** [comment]
I believe the method `LoadConfig`, `LoadDefaultConfig` can also be removed. If we want to use the metadata/process class stand-alone we instantiate it and call `LoadConfigFromFile`, which is provided by `TRestMetadata`. I think it is worthy to do so. We can simplify a lot of code for the processes.
